js/top5parser.js

```diff
@@ -0,0 +1,42 @@
+function updateTopArticles() {
+
+  console.log('refreshing top articles');
+
+  var url = 'https://ubyssey-analytics.appspot.com/query?id=ahNzfnVieXNzZXktYW5hbHl0aWNzchULEghBcGlRdWVyeRiAgICA7a2SCgw&format=json';
+
+  $.ajax({
+    type: 'GET',
+    url: url,
+    dataType: 'jsonp',
+    success: function(data) {
+      renderHTML(data.rows);
+    }
+  });
+
+  function renderHTML(data) {
+    var date = new Date(Date.now());
+    var hour = date.getHours();
+    var title = '';
+    var views = '';
```

Comment:
You can just declare these on one line like this:

```js
var title, views;
```

No need to set them as empty strings.